flink-runtime/src/main/scala/org/apache/flink/runtime/jobmanager/JobManager.scala

```diff
@@ -327,8 +327,11 @@ class JobManager(
       currentJobs.get(jobID) match {
         case Some((executionGraph, jobInfo)) => executionGraph.getJobName
 
+          val deserializedError = if(error != null) {
+            error.getError(executionGraph.getUserClassLoader)
```

Comment:
why not renaming `error.getError` => `serializedError.deserializeError`? Makes things a little bit clearer.